.github/workflows/deploy-to-control-plane.yml

```
@@ -207,13 +329,43 @@ jobs:
         if: env.DO_DEPLOY != 'false'
         run: git checkout ${{ steps.getRef.outputs.PR_REF }}
 
+      - name: Initialize GitHub Deployment
+        if: env.DO_DEPLOY != 'false'
+        uses: actions/github-script@v7
+        id: init-deployment
+        with:
+          script: |
+            const ref = process.env.PR_SHA;
+            const environment = process.env.ENVIRONMENT_NAME || 'review-app';
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)